content/jobs/2023/23qQ4-open-source-infrastructure-engineer.md

```diff
@@ -0,0 +1,116 @@
++++
+authors =  ["Damián Avila"]
+title = "Open Source Infrastructure Engineer"
+work_description = [
+  "Cloud infrastructure management and operations",
+  "Site Reliability Engineering",
+  "Development of open source infrastructure for hosted JupyterHub service"
+]
+salary_range = "$121,600 - $130,500"
+location = "Remote, required overlapping for US Central and US Pacific timezones"
+date = "2022-10-10"
+url_apply = "https://forms.gle/k75VmYUosxuyPWfi7"
+deadline = "We will begin reviewing applications around **October 16th**, and will accept them on a rolling basis until the position is filled."
+open = true
+show_date = true
++++
+
+We are looking for an experienced Open Source Infrastructure Engineer who will help shape the future
+of data-intensive scientific research and make a big impact on democratizing the design and access
+to cloud-based resources for research and education purposes. This engineer will be part of an awesome
+engineering team pushing forward the development and reliable operations of our cloud-based infrastructure.
+
+<!-- Defined in layouts/shortcodes/job_details.html -->
+{{% job_details %}}
+
+## Who we are
+
+2i2c is a non-profit organization with a [mission](https://2i2c.org/mission/) to make interactive computing
+more accessible, scalable, and powerful for research and education.
+
+We accomplish this mission by providing managed cloud services for interactive computing, by providing
+development and technical leadership to researchers and educators that utilize this infrastructure for
+specific communities, and by providing support for open source tools and communities in this ecosystem.
+
+We have deep ties to the open source community and have been leaders and core contributors across dozens
+of projects - in particular in the Jupyter ecosystem. We also have deep ties to research and education -
+our team has spent years deploying infrastructure for universities, community colleges, and research teams,
+and now we’re bringing this experience to a wider audience with 2i2c.
+
+We believe strongly in communities that are inclusive, transparent, equitable, effective, and diverse,
+especially 2i2c itself. We believe that our values should permeate everything about 2i2c, including the
+work we do, the communities we serve, and our own organizational culture.
+
+2i2c is a 501(c)(3) not-for-profit project fiscally sponsored by Code for Science and Society.
+
+
+## What you’ll do
+
+- Ensures the reliable operation of the 2i2c infrastructure (leveraging production-ready cloud-based
+  tools such as JupyterHub, BinderHub, and Dask).
+- Attends support-facing tickets
```

Comment:
```suggestion
- Attends support-facing tickets as part of a support rotation process/group (this is not your _primary_ task, it is currently a shared load among engineers).
```

Reply:
I tried to clarify it, we can change it after it is published if it is not enough (or if there is a better version).